client/modules/users/authReducer.js

```diff
@@ -30,6 +36,32 @@ export const clearUser = () => ({type: CLEAR_USER})
 
 export const clearFlags = () => ({type: CLEAR_FLAGS})
 
+export const loadNotifications = () => ({
+  [CALL_API]: {
+    endpoint: '/users/me/listnotifications',
```

Comment:
Couldn't these all use the same endpoint, `users/me/notifications/:id?`? GET should return all notifications, DELETE could delete by id or delete all if no id is passed. I'm not sure why you'd need to update a notification so the setProfileNotifications function below could probably be removed